code/game/objects/items/weapons/twohanded.dm

```diff
@@ -562,43 +566,41 @@
 /obj/item/twohanded/singularityhammer/process()
 	if(charged < 5)
 		charged++
-	return
 
 /obj/item/twohanded/singularityhammer/update_icon()  //Currently only here to fuck with the on-mob icons.
 	icon_state = "mjollnir[wielded]"
-	return
-
-/obj/item/twohanded/singularityhammer/proc/vortex(var/turf/pull as turf, mob/wielder as mob)
-	for(var/atom/X in orange(5,pull))

```

Comment:
Why this looped through every single atom, then did a type check for `/atom/movable` is beyond me. Why not just do built in type checking by making it `/atom/movable` to begin with?